README.md

```diff
@@ -0,0 +1,19 @@
+## Apollo Product README
+
+Apollo Product its a software product tool to help leadership guide the vision and delivery of the team.
```

Comment:
Maybe "Apollo is a software product tool to help leadership guide the vision and delivery of the team"